cmd/openshift-install/gather.go

```diff
@@ -134,16 +155,16 @@ func extractHostAddresses(config *types.InstallConfig, tfstate *terraform.State)
 	case openstacktypes.Name:
 		bootstrap, err = gatheropenstack.BootstrapIP(tfstate)
 		if err != nil {
-			return bootstrap, masters, err
+			return bootstrap, port, masters, err
 		}
 		masters, err = gatheropenstack.ControlPlaneIPs(tfstate)
 		if err != nil {
 			logrus.Error(err)
 		}
 	default:
-		return "", nil, errUnSupportedGatherPlatform{Message: fmt.Sprintf("Cannot fetch the bootstrap and control plane host addresses from state file for %s platform", config.Platform.Name())}
+		return "", port, nil, errUnSupportedGatherPlatform{Message: fmt.Sprintf("Cannot fetch the bootstrap and control plane host addresses from state file for %s platform", config.Platform.Name())}
 	}
-	return bootstrap, masters, nil
+	return bootstrap, port, masters, nil
```

Comment:
The default case above would ensure that this line is never reached, right? If so is there a better way of handling that? Should it at least include an error?